temporian/core/operators/calendar/test/test_timezone_argument.py

```diff
@@ -0,0 +1,126 @@
+# Copyright 2021 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import datetime, timedelta, timezone
+from absl.testing import absltest
+from absl.testing.parameterized import TestCase
+
+from temporian.implementation.numpy.data.io import event_set
+from temporian.test.utils import assertOperatorResult, i32
+
+
+class CalendarTimezoneTest(TestCase):
```

Comment:
I feel these tests might be easier to miss here than if they were in each operator's test file - is there a reason to have them here together?

Reply:
The reason is I don't want to duplicate the very specific setUp() code, and also it's easier to check that I'm testing that the argument is connected in all operators.
I agree it's easy to miss the file though, so I renamed it to `test_base_timezone` and now it appears on top together with `test_base.py` (which already existed and tests other generic stuff).